### PR TITLE
Control flow style

### PR DESCRIPTION
**Pull request by Mixpa**
simplify code

### CODE REVIEW

File: src/main/java/org/apache/commons/io/IOUtils.java [hunk 1]
```diff
@@ -778,11 +778,11 @@ public static boolean contentEqualsIgnoreEOL(final Reader input1, final Reader i
 
         String line1 = br1.readLine();
         String line2 = br2.readLine();
-        while (line1 != null && line2 != null && line1.equals(line2)) {
+        while (line1 != null && line1.equals(line2)) {
             line1 = br1.readLine();
             line2 = br2.readLine();
         }
-        return line1 == null ? line2 == null ? true : false : line1.equals(line2);
+        return line1 == null && line2 == null;
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/io/IOUtils.java]: What about the `line1.equals(line2)` bit?

Comment (Mixpa) [reply src/main/java/org/apache/commons/io/IOUtils.java]: yes,  you way is better than me

File: src/main/java/org/apache/commons/io/IOUtils.java [hunk 2]
```diff
@@ -778,11 +778,11 @@ public static boolean contentEqualsIgnoreEOL(final Reader input1, final Reader i
 
         String line1 = br1.readLine();
         String line2 = br2.readLine();
-        while (line1 != null && line2 != null && line1.equals(line2)) {
+        while (line1 != null && line1.equals(line2)) {
             line1 = br1.readLine();
             line2 = br2.readLine();
         }
-        return line1 == null ? line2 == null ? true : false : line1.equals(line2);
+        return line1.equals(line2);
```
Comment (Stzx) [inline src/main/java/org/apache/commons/io/IOUtils.java]: line1 may be null.

Comment (Mixpa) [reply src/main/java/org/apache/commons/io/IOUtils.java]: how about Objects.equals

**Comment by Mixpa** [comment]
@garydgregory

**Pull request merged**